kani-compiler/src/main.rs

```diff
@@ -65,12 +70,19 @@ fn rustc_gotoc_flags(lib_path: &str) -> Vec<String> {
         "crate-attr=feature(register_tool)",
         "-Z",
         "crate-attr=register_tool(kanitool)",
+        // Prints expanded macro. For proptest devops only, remove after done
```

Comment:
Do we want to comment this out?

Reply:
I'll remove this. When this is enabled, it seems something breaks and we no longer generate required files. Also, it is pretty noisy for larger libraries.

Reply:
Purged with a66a89c5beac3f8657be02d961073083c7b2e094